sbt-test/sbt-dotty/dotty-knowledge.i17/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.8.2
+sbt.version=1.9.3
```

Comment:
I think all those `build.properties` files are useless because sbt uses the `sbtVersion` key to run the scripted tests.

Reply:
A good call, I'll remove the ones that are in the scripted tests.